ovos_utils/configuration.py

```diff
@@ -42,59 +42,70 @@ def get_xdg_cache_save_path(folder=None):
     return join(xdg_cache_home(), folder)
 
 
+def _is_running_from_module(module_name):
+    #is_installed = find_spec(module_name) is not None
+    #if not is_installed:
+    #    return False
+    in_path = any([p.endswith(f"/{module_name}") for p in sys.path])
```

Comment:
previously if you had hivemind voice satellite installed it would load hivemind.conf instead of mycroft.conf

this is still not an optimal solution for when multiple cores exist in the same .venv ....

this change also allows detecting ovos-core vs mycroft (both show up as mycroft in find_spec)

Reply:
get_distribution is another way to get package name, rather than import name https://github.com/NeonGeckoCom/neon-skill-utils/blob/95c3a1225a403336f5ad89bf0e7b9509af373820/neon_utils/packaging_utils.py#L68